### ISSUE TITLE
Crash on non existant indes. 

### ISSUE THREAD

**Issue by alekc** [phpWhois/phpWhois#49]
There is a constant crash during parsing of ip 52.48.87.87

It's caused in whois.ip.php
`list($low,$high) = explode('-',str_replace(' ','',substr($line,$p+strlen($net)+3)));`

The issue is caused by following line of text
`Parent:         NET52 (NET-52-0-0-0-0)`

In this case explode returns only 1 result so the whole line will fail.

**Comment by mcormick** [comment]
Yeah, discussion at: https://github.com/phpWhois/phpWhois/issues/8
